addon/integrations/google-analytics.js

```diff
@@ -59,8 +59,13 @@ export default Base.extend({
     }
 
     for (let key in options) {
-      const capitalizedKey = capitalize(key);
-      gaEvent[`event${capitalizedKey}`] = options[key];
+      const value = options[key];
+
+      // If key is not a 'dimension' or 'metric', prepend with 'event'
+      let shouldPrefix = !/^(dimension|metric)[0-9]{1,2}/.test(key);
```

Comment:
While the variable is not strictly necessary, I believe it improves readability (as opposed to having the test directly in the `if`` statement).